j/jekyll-theme-midnight/jekyll-theme-midnight_ubi_8.5.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : jekyll-theme-midnight
+# Version       : v0.1.1
+# Source repo	: https://github.com/pages-themes/midnight
+# Tested on     : UBI 8.5
+# Language      : Ruby
+# Travis-Check  : False
```

Comment:
Can you please state the reason for skipping travis-check ?

Reply:
the build was failing on travis , it had some utf8 encoding error which couldn't be resolved , so I was told to change the travis check to False since the build was successful on the local container . Had a similar issue with the other 2 Jekyll-theme packages

Reply:
Thanks for the udpates.